test/libsolidity/smtCheckerTests/functions/getters/external_getter_this_2.sol

```diff
@@ -0,0 +1,18 @@
+contract C {
```

Comment:
Does this test fail without the change in the getter? Ah, yeah, it probably does - but I guess that ``assert(y == 0);`` was what would have passed without the change? Just for verifying that I got this stuff right - change and test look fine otherwise.

Reply:
Without this commit's code the getter would just behave the same as an external call to unknown code and return something nondeterministic.

Reply:
Which is a massive false positive since we actually know the address of `c`. This code just makes it stronger.